libs/cdk/internals/scheduler/src/lib/scheduler.ts

```diff
@@ -71,18 +67,10 @@ const clearTimeout = ɵglobal.clearTimeout;
 const setImmediate = ɵglobal.setImmediate; // IE and Node.js + jsdom
 const messageChannel = ɵglobal.MessageChannel;
 
-const isInputPending =
-  typeof ɵglobal.navigator !== 'undefined' &&
-  ɵglobal.navigator.scheduling !== undefined &&
-  ɵglobal.navigator.scheduling.isInputPending !== undefined
-    ? ɵglobal.navigator.scheduling.isInputPending.bind(
-        ɵglobal.navigator.scheduling
-      )
-    : null;
-
```

Comment:
this is actually something we want to keep.

Reply:
Do we want to keep something not used? Why?

Reply:
because it is part of the roadmap to test this API :D

Reply:
Then we could introduce it when we need it.

Reply:
okay, fine by me